.github/workflows/getBuildLists.py

```diff
@@ -15,7 +16,19 @@ def main():
             result.append('functional')
         elif (argument.startswith('openjdk/') and ('openjdk' not in result)):
             result.append('openjdk')
-
+        if argument.startswith('buildenv/') and not flag:
+            print('::set-output name=build_env::true')
+            flag = True
+    
+    if not flag:
+        print('::set-output name=build_env::false')
+    
+    if len(result) > 0:
+        print('::set-output name=test_dirs_changed::true')
+    else:
+        print('::set-output name=test_dirs_changed::false')
```

Comment:
This could be combined with 

```
 if not result:
        result.append('skip')	        result.append('skip')
```

and  update the if condition in https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/.github/workflows/directoriesFilesChangePR.yml#L32 ?

Reply:
Good point! I've revised it now.